packages/host/config/schema/1759232510123_schema.sql

```diff
@@ -55,6 +55,14 @@
    PRIMARY KEY ( url, realm_url ) 
 );
 
+ CREATE TABLE IF NOT EXISTS claimed_domains_for_sites (
+   id DEFAULT (hex(randomblob(16))) NOT NULL,
+   hostname TEXT NOT NULL,
```

Comment:
Using `hostname` for the following reason:
1. Being able to perform a join with published_realms on `source_realm_url` so that we will be able to reference the published realm
2. We can use this for another purpose - when users bring their own domains (planned future feature)